luma/core/cmdline.py

```diff
@@ -300,14 +306,18 @@ def create_parser(description):
     ftdi_group = parser.add_argument_group('FTDI')
     ftdi_group.add_argument('--ftdi-device', type=str, default='ftdi://::/1', help='FTDI device')
 
+    linux_framebuffer_group = parser.add_argument_group('LINUX_FRAMEBUFFER')
```

Comment:
'Linux framebuffer' sounds/looks better to me.

Reply:
done